controllers/cart.controller.js

```diff
@@ -0,0 +1,114 @@
+const Cart = require("../models/cart.model");
+const User = require("../models/user.model");
+
+const getCarts = async (req, res) => {
+  const carts = await Cart.find({});
+  res.json({ success: true, carts });
```

Comment:
It would be great if we use trycatch here also

Reply:
As Tanay mentioned that the common error handler could be used in most of the cases except where i need custom errors, hence i tried to implement the same.